### PR TITLE
feat(proguard): Release association

### PR DESCRIPTION
**Pull request by buenaflor**
Creates a weak release association with a proguard mapping file. https://github.com/getsentry/team-mobile/issues/27

### CODE REVIEW

**Review by loewenheim — APPROVED** [review]
Looks good, apart from lint stuff :)

File: src/api.rs [hunk 1]
```diff
@@ -1354,6 +1354,28 @@ impl Api {
...
+        let path = format!(
+            "/projects/{}/{}/files/proguard-artifact-releases",
+            PathArg(org),
+            PathArg(project)
+        );
+        let resp: ApiResponse = self
+            .request(Method::Post, &path)?
+            .with_json_body(data)?
+            .send()?;
+        if resp.status() == 404 || resp.status() == 201 {
```
Comment (kamilogorek) [inline src/api.rs:1372]: Why are we doing `Ok(())` on `404`? 👀

File: src/api.rs [hunk 2]
```diff
@@ -1361,16 +1361,18 @@ impl Api {
         data: &AssociateProguard,
     ) -> ApiResult<()> {
         let path = format!(
-            "/projects/{}/{}/files/proguard-artifact-releases",
+            "/projects/{}/{}/files/proguard-artif",
```
Comment (kamilogorek) [inline src/api.rs:1364]: Whoops

File: src/api.rs (hunk 1 above)
Comment (buenaflor) [reply src/api.rs]: Shouldn't be there for sure!

**Review by kamilogorek — APPROVED** [review]

**Pull request merged**